app/models/configuration_script_base.rb

```diff
@@ -0,0 +1,15 @@
+class ConfigurationScriptBase < ApplicationRecord
```

Comment:
@bdunne If we are changing this, do we need a data migration for any polymorphic reference?